src/services/assessmentHooks.ts

```diff
@@ -183,6 +183,9 @@ export const useUpdateSurveyAssessment = () => {
 
       case 'UPDATE':
         console.log('updating', assessment)
+        if (assessment.isReadOnly) {
+          throw Error('Cannot update a published assessment')
+        }
```

Comment:
Unfortunately, no. I asked. Apparently it is only the *config* where it will throw an error if you make changes to a published assessment for ...reasons.